releng/org.eclipse.xtext.sdk.p2-repository/pom.xml

```diff
@@ -13,6 +13,15 @@
 	<artifactId>org.eclipse.xtext.sdk.p2-repository</artifactId>
 	<version>2.26.0-SNAPSHOT</version>
 	<packaging>eclipse-repository</packaging>
+	
+	<properties>
+		<!-- 
+			Version qualifier of the target zip file 
+			Usually this is the same as the (snaphsot) project version
+			but for releases it is set to the release version.
+		-->
+		<zipFileVersion>2.26.0-SNAPSHOT</zipFileVersion>
```

Comment:
can the version adjustment handle this one?

Reply:
I'll take care of it. This is next on my list.